src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,11 @@
+.UserInfo {
+  text-decoration: none;
+  text-transform: none;
+  font-size: 18px;
+  color: #333;
+  transition: color 0.4s ease-in-out;
+
+  &:hover {
+    color: #a36f6f;
```

Comment:
Consider using a more descriptive color name or variable instead of `#a36f6f`. 🎨